libs/ngx-dsfr/navigation/src/lib/navigation.component.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Angular imports
+ */
+import { Component, ElementRef, Input, OnInit, QueryList, ViewChildren } from '@angular/core';
+import { Router } from '@angular/router';
+
+/**
+ * 3rd Party Imports
+ */
+import { UserRole } from '@betagouv/ngx-dsfr';
+
+/**
+ * TypeScript entities and constants
+ */
+export const EMPTY_NAVIGATION_ERROR: string =
+  'You MUST provide a navigation data for this component 😡 !!!';
```

Comment:
`'You MUST provide navigation data for this component 😡 !!!';`

Data is not countable in English